test/reporter/test-reporter-util.js

```diff
@@ -79,6 +79,7 @@ test('util.sanitizeOutput', function (t) {
   var raw = fs.readFileSync(carriageReturnPath, 'utf-8');
   var expected = fs.readFileSync(carriageReturnExpectedPath, 'utf-8');
   var result = util.sanitizeOutput(raw, '#');
-  t.equals(result, expected, 'there should be a # on every line');
+  result += '\n';
```

Comment:
I think some Unix editors require a newline at the end of a file, which (I think) is why Node's [.editorconfig](https://github.com/nodejs/node/blob/master/.editorconfig#L7) sets the `newline at the end of file` option. It might make sense to standardise on that, in which case we'd change the test. Of course we might not want to, I don't know.

Possibly relevant: http://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline